tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java

```diff
@@ -202,7 +256,7 @@ private KafkaProducer<byte[], byte[]> createProducer(Properties config) {
     }
 
     private Consumer<byte[], byte[]> createConsumer(Properties config) {
```

Comment:
We can change the type from `Properties` to `Map<String, Object>`. With that change we don't need to create a lot of `Properties` in each test case

Reply:
Updated it and also added empty map for zk, or zk will not be tested.